make/hotspot/lib/JvmFeatures.gmk

```diff
@@ -128,10 +129,6 @@ ifneq ($(call check-jvm-feature, cds), true)
       heapShared.cpp \
       lambdaFormInvokers.cpp \
       metaspaceShared.cpp \
-      metaspaceShared_$(HOTSPOT_TARGET_CPU).cpp \
-      metaspaceShared_$(HOTSPOT_TARGET_CPU_ARCH).cpp \
-      sharedClassUtil.cpp \
-      sharedPathsMiscInfo.cpp \
```

Comment:
Removed obsolete files that no longer exist.

Reply:
Thank you!

Reply:
I don't suppose we can just exclude the new directory rather than listing individual files?

Reply:
Fixed. Now all files under share/cds are excluded. I needed to move compactHashtable.cpp back to its old location since a little of it is used even when CDS is not used.

Reply:
That's indeed a better idea.